### DISCUSSION TITLE
how to remove this warning in fluentd? [warn]: #0 Detected ES 8.x or above: document type will not be used

### DISCUSSION THREAD

**Discussion by sungwon0** [fluent/fluentd#4132] [Q&A]
I was running well with efk stack. 
Recently I upgraded elasticsearch 7.x to  elasticsearch 8.x , there have been 70 warnings per second.
( [warn]: #0 Detected ES 8.x or above: document type will not be used) 
how to resolve this problem?
My efk environment :
- elasticsearch 8.6 
- td-agent 3.8.1
- os : windows

**Comment by daipom** [comment]
I don't know well about this message, but it seems to be already fixed at [fluent-plugin-elasticsearch](https://github.com/uken/fluent-plugin-elasticsearch) 4.3.2

* https://github.com/uken/fluent-plugin-elasticsearch/pull/852

You should try the latest td-agent.
